src/errors/errors.enum.ts

```diff
@@ -7,7 +7,9 @@ enum ErrorsTypes {
   NOT_FOUND_ACCOMMODATION_FOR_DELETING = 'NOT_FOUND_ACCOMMODATION_FOR_DELETING',
   NOT_FOUND_ACCOMMODATION_FOR_RESTORING = 'NOT_FOUND_ACCOMMODATION_FOR_RESTORING',
   NOT_FOUND_ACCOMMODATION = 'NOT_FOUND_ACCOMMODATION',
+  NOT_ENOUGH_IMAGES_TO_UPLOAD = 'NOT_ENOUGH_IMAGES_TO_UPLOAD',
```

Comment:
it would be better if you rename this to somehing like BAD_REQUEST_NOT_ENOUGH_IMAGES_TO_UPLOAD to keep naming consistency